px-map-behavior-popup.es6.js

```diff
@@ -247,8 +247,9 @@
 
       const maxWidth = 400;
       const minWidth = 300;
+      const maxHeight = 300;
```

Comment:
Yep, go for it. If you want you can take this code out of this PR and we can merge the marker double click stuff right away, or just do it all together. Up to you.